ci/pipeline.yml

```diff
@@ -129,8 +129,8 @@ resources:
 - name: bosh-src
   type: git
   source:
-    uri: https://github.com/jianqiu/bosh.git
-    branch: 256.x-softlayer
+    uri: https://github.com/zhanggbj/bosh.git
```

Comment:
For stemcell, currently, we still have to fork due to the following reasons:

1. Need to enable root ssh remote login, both Bluemix and eCPI require it.
2. For Bluemix stemcell, we need to install openssl 1.0.2 and fips module.